build/PackageArchive.targets

```diff
@@ -0,0 +1,78 @@
+<Project>
+  <PropertyGroup>
+    <_TemplatesDirectory>$(MSBuildThisFileDirectory)tools\templates\</_TemplatesDirectory>
+    <_WorkRoot>$(RepositoryRoot).w\</_WorkRoot>
+  </PropertyGroup>
+
+  <Target Name="_BuildFallbackArchive" DependsOnTargets="ResolveRepoInfo">
+    <Error Text="DotNetRestoreSources must be specified" Condition=" '$(DotNetRestoreSources)' == '' " />
+    <Error Text="OutputPackageName must be specified" Condition=" '$(OutputPackageName)' == '' " />
+    <Error Text="ArchiverPath must be specified" Condition=" '$(ArchiverPath)' == '' " />
```

Comment:
What is ArchiverPath and how do I get it? We should use existing build tools or commit them to source where possible.

Reply:
There are plans on moving the tool to this repo. I'll file an issue for this. Currently it just lives in our drop share. Since we want to get this done ASAP I'm not bothering with moving it for now.

Reply:
👍 for filing an issue. Thanks.

Reply:
https://github.com/aspnet/Universe/issues/595